docs/architecture/architecture.md

```diff
@@ -609,6 +614,133 @@ rootfs by calling `mount(8)` within the counter.  If the devicemapper block devi
+orchestration support was an afterthought with 2.1, and that supporting the single container
+Docker use case was the main focus.
+
+Starting from the 3.0 release, Clear Containers took a different, almost opposite, approach.
+The Clear Containers 3.0 runtime is based on the `virtcontainers` package whose API is
+inspired by the Kubernetes `CRI` one. `virtcontainers` execution units are pods and it does
+not create or manage any container outside of an existing pod context, and thus so does the
+Clear Container 3.0 runtime.
+To create a Docker container it will first create a pod which will represent the container
+hardware virtualized context and then start an actual container inside that virtual machine.
+Both the pod and its container have the same identifier and name.
+
+### Mixing VM based and namespace based runtimes
+
+One interesting evolution of the `CRI-O` support for the Clear Containers runtime is the ability
+to run virtual machine based pods alongside namespace ones. With `CRI-O` and Clear Containers,
+one can introduce the concept of workload trust inside a Kubernetes cluster.
+
+A cluster operator can now tag (through Kubernetes annotations) container workloads as `trusted`
+or `untrusted`. The former labels known to be safe workloads while the latter describes
+potentially malicious or misbehaving workloads that need the highest degree of isolation.
+In a software development context, an example of a `trusted` workload would be a containerized
+continuous integration engine whereas all developers applications would be `untrusted` by default.
+Developers workloads can be buggy, unstable or even include malicious code and thus from a
+security perspective it makes sense to tag them as `untrusted`. A `CRI-O` and Clear Containers
+based Kubernetes cluster handles this use case transparently as long as the deployed containers
+are properly tagged. All `untrusted` containers will be handled by Clear Containers and thus run
+in a hardware virtualized secure sandbox whilst `runc`, for example, could  handle the
+`trusted` ones.
+
+`CRI-O`'s default behaviour is to trust all pods, except when they're annotated with
+`io.kubernetes.cri-o.TrustedSandbox` set to `false`. The default `CRI-O` trust level is
+set through its `configuration.toml` configuration file. Generally speaking, the `CRI-O`
+runtime selection between its trusted runtime (typically `runc`) and its untrusted one
+(Clear Containers) is a function of the pod `Privileged` setting, the `io.kubernetes.cri-o.TrustedSandbox`
+annotation value, and the default CRI-O trust level. When a pod is `Privileged`, the
+runtime will always be `runc`. However, when a pod is **not** `Privileged` the runtime
+selection is done as follows:
+
+|                                          | `io.kubernetes.cri-o.TrustedSandbox` = not set | `io.kubernetes.cri-o.TrustedSandbox` = `true` | `io.kubernetes.cri-o.TrustedSandbox` = `false` |
```

Comment:
miniscule nit: I'd remove the "=" so that column just shows, "`io.kubernetes.cri-o.TrustedSandbox` not set".